packages/page-template-modal/package.json

```diff
@@ -0,0 +1,56 @@
+{
+	"name": "@automattic/page-template-modal",
+	"version": "1.0.0",
```

Comment:
This is version `1.0.0`, but ETK depends on version `1.0.0-alpha.0`. They should be the same version, right?